src/makeInfinityAdder.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  const adder = (num) => {
+    if (num === undefined) {
+      return 0;
+    }
```

Comment:
You don't need an additional function (adder) just to handle this case
Just define the initial `sum` as `0`

Note that in this case, you will need to handle resetting sum before returning it